README.md

```diff
@@ -1,707 +1,1799 @@
+a single ["result function"][`result function`] / ["combiner"][`combiner`], and an optional options object, and
+generates a memoized selector function.
+
+</details>
+
+<details><summary><b>Parameters</b></summary>
+
+| Name                     | Description                                                                                      |
+| :----------------------- | :----------------------------------------------------------------------------------------------- |
+| `inputSelectors`         | An array of [`input selectors`], can also be passed as separate arguments.                       |
+| `combiner`               | A [`combiner`] function that takes the results of the [`input selectors`] as separate arguments. |
+| `createSelectorOptions?` | An optional options object that allows for further customization per selector.                   |
+
+</details>
+
+<details><summary><b>Returns</b></summary>
+
+A memoized [`output selector`].
+
+</details>
+
+<details><summary><b>Type parameters</b></summary>
+
+| Name                          | Description                                                                                                                                                                                            |
+| :---------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `InputSelectors`              | The type of the [`input selectors`] array.                                                                                                                                                             |
+| `Result`                      | The return type of the [`combiner`] as well as the [`output selector`].                                                                                                                                |
+| `OverrideMemoizeFunction`     | The type of the optional `memoize` function that could be passed into the options object to override the original `memoize` function that was initially passed into [`createSelectorCreator`].         |
+| `OverrideArgsMemoizeFunction` | The type of the optional `argsMemoize` function that could be passed into the options object to override the original `argsMemoize` function that was initially passed into [`createSelectorCreator`]. |
+
+</details>
+
+<div align="right">[ <a href="#table-of-contents">↑ Back to top ↑</a> ]</div>
+
+---
+
+### Memoization Functions
+
+<a id="defaultmemoize"></a>
+
+#### defaultMemoize(func, equalityCheckOrOptions = defaultEqualityCheck)
+
+<details><summary><b>Description</b></summary>
+
+The standard memoize function used by [`createSelector`].
+
+It has a default cache size of 1. This means it always recalculates when the value of an argument changes. However, this can be customized as needed with a specific max cache size (**`Since`** 4.1.0).
```

Comment:
Don't need to format "since" anywhere:

```suggestion
It has a default cache size of 1. This means it always recalculates when the value of an argument changes. However, this can be customized as needed with a specific max cache size (since 4.1.0).
```

Reply:
Initially got that from [markdown-magic](#https://github.com/DavidWells/markdown-magic). But you're right I will fix it :)